ui/server/src/main/scala/pl/touk/nussknacker/ui/validation/ProcessValidation.scala

```diff
@@ -186,9 +186,21 @@ class ProcessValidation(validators: ProcessingTypeDataProvider[ProcessValidator]
   }
 
   private def validateWithCustomProcessValidator(process: DisplayableProcess): ValidationResult = {
```

Comment:
I wonder, maybe it would be better to just move CustomProcessValidatorLoader to interpreter and invoke it in ProcessValidator?